samples/languages/nodejs/README.md

```diff
@@ -0,0 +1,20 @@
+Running NodeJS with SGX-LKL
+===========================
+
+1. Build a Docker container with NodeJS:
+```
+$ docker build -t nodejs .
+```
+
+2. Convert the container to an SGX-LKL root file system image:
+```
+$ sgx-lkl-disk create --docker=nodejs --size=100M nodejs.img
+```
+
+3. Run NodeJS demo program with SGX-LKL:
+```
+$ SGXLKL_MMAP_FILES=Shared sgx-lkl-run-oe --sw-debug --host-config=nodejs-host_config.json --enclave-config=nodejs-enclave_config.json 
```

Comment:
Any particular reason why you choose to have prefixed filenames?

Reply:
I find it clearer to have the host/enclave config filenames to be specific to the application.